devbot/database.py

```diff
@@ -11,21 +11,19 @@
 dotenv.load_dotenv(".env")
 
 # Set database URL or use SQLite if none is given
-db_url = (
-    os.environ.get("DATABASE")
-    if os.environ.get("DATABASE")
-    else "sqlite:///database.sqlite"
-)
+DB_URL = os.environ.get("DATABASE", "sqlite:///database.sqlite")
 
-SQLAlchemyBase = declarative_base()
-engine = sa.create_engine(db_url, echo=False)
-Session = sessionmaker(bind=engine)
+SQLAlchemyBase = declarative_base()  # pylint: disable=invalid-name
+ENGINE = sa.create_engine(DB_URL, echo=False)
+Session = sessionmaker(bind=ENGINE)  # pylint: disable=invalid-name
 
 
 class User(SQLAlchemyBase):
+    """ Representation of a seen Discord user, unused. """
+
     __tablename__ = "user"
     # https://discordapp.com/developers/docs/resources/user
-    id = sa.Column(sa.String(32), primary_key=True)
+    id = sa.Column(sa.String(32), primary_key=True)  # pylint: disable=invalid-name
```

Comment:
Since id is a reserved keyword it would be better to change it instead of telling pylint to shut up.